lib/ca_testing/logger.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require "logger"
+
+module CaTesting
+  #
+  # @api private
+  #
+  class Logger
+    def create(output = $stdout)
```

Comment:
Maybe a class method to avoid having to do `Logger.new.create`.  `CaTesting::Logger` isn't really a class as it is never instantiated.  Not sure what a typically pattern is in these cases.

```suggestion
    def self.create(output = $stdout)
```